### DISCUSSION TITLE
How to highlight hyperlinks?

### DISCUSSION THREAD

**Discussion by WuSiren** [sumatrapdfreader/sumatrapdf#4625] [Documentation]
I installed the latest version of SumatraPDF (3.5.2) and I found that all the hyperlinks in a paper are not highlighted. But in an older version I used, they were clearly highlighted. 
![image](https://github.com/user-attachments/assets/5e7e63d5-ed1c-41a0-8aed-250d846e969a)

**Comment by GitHubRulesOK** [comment]
The blue highlighting around hyperlinks is usually in the debug/pre-release versions however one release include those options
![image](https://github.com/user-attachments/assets/dee4e1bc-0f04-49b9-afe0-a01c3ce433f5)

**Comment by WuSiren** [comment]
I found that v3.1.2 64bit supports this by default (i.e. cross-references, urls, etc., can be displayed in different colors). I don't know why later versions dropped this feature, I think it's very useful, and I don't know how to get this feature in the new version.

**Reply by GitHubRulesOK** [reply]
It has usually been classed as "experimental" so in the debug section and not intended as "release" ready
The current [PRE-release](https://www.sumatrapdfreader.org/prerelease) has it there as bright blue and perhaps debug shown in one other more released version ? 3.2 ?
